File: Iniciante/1012 - Área/C/1012 - Área.c
#include <stdio.h>
 
int main() {
	double A, B, C; // variáveis
	
	scanf("%lf %lf %lf", &A, &B, &C); // recebe os dados de entrada
	
	printf("TRIANGULO: %.3f\n", (A*C)/2);
	printf("CIRCULO: %.3f\n", 3.14159*(C*C));
	printf("TRAPEZIO: %.3f\n", ((A+B)*C)/2);
	printf("QUADRADO: %.3f\n", B*B);
	printf("RETANGULO: %.3f\n", A*B); // apresenta o resultado e realiza os cálculos
	
	return 0;
}

File: Iniciante/1012 - Área/Python/1012 - Área.py
# pega 3 valores na mesma linha e atribui às variáveis
A, B, C = input().split(' ')

# Converte os valores para os tipos necessários 
A = float(A)
B = float(B)
C = float(C)

# cálculos aritméticos
triangulo = (A*C)/2
circulo = 3.14159*(C**2)
trapezio = ((A+B)*C)/2
quadrado = B**2
retangulo = A*B

# saída
print('TRIANGULO: {:.3f}'.format(triangulo))
print('CIRCULO: {:.3f}'.format(circulo))
print("TRAPEZIO: {:.3f}".format(trapezio))
print('QUADRADO: {:.3f}'.format(quadrado))
print('RETANGULO: {:.3f}'.format(retangulo))